### ISSUE TITLE
Missing text label on system tray icon makes it harder for screen reader users to find it

### ISSUE THREAD

**Issue by Neurrone** [docker/for-win#12460]
-[x] I have tried with the latest version of Docker Desktop
  -[x] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Actual behavior

The docker system tray icon does not have text associated with it. This means that screen reader users can't find the button easily, because it just gets reads out as "button".

However, when Docker needs to be restarted to apply an update, this label does get added to the system tray icon: I presume this is what would appear when mousing over the icon: "An update will be applied next time you start Docker Desktop"

### Expected behavior

The system tray icon always gets a label associated with it, not just when Docker needs to be restarted to update it.

### Information

  - Windows Version: Windows 10 21h2
  - Docker Desktop Version: 4.3.1
  - WSL2 or Hyper-V backend? WSL2 
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: no

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by lorenrh** [comment]
Thanks for the report, I've created an internal ticket for this!

**Comment by Neurrone** [comment]
Can confirm this is fixed in the latest v4.6.0 version of Docker.
